source/module_hamilt_lcao/module_deepks/LCAO_deepks_vdelta.cpp

```diff
@@ -5,10 +5,6 @@
 //tr (rho * V_delta)
 
 //Four subroutines are contained in the file:
-//1. add_v_delta : adds deepks contribution to hamiltonian, for gamma only
```

Comment:
Could you explain why these lines and related functions were deleted?

Reply:
A brief look at the code would reveal that this part of functionality has been long ago replaced in some previous reconstruction.